demo/db.json

```diff
@@ -47,6 +47,30 @@
 		"invalid": true,
 		"functional_group": "Vacuum",
 		"location_group": "S 2"
+	},
```

Comment:
I don't know where this came from.

Reply:
This is normal json, the diff just looks a little funny

Reply:
git decided that this was the new `}` instead of the trailing one on line 74